legacy/flash.h

```diff
@@ -45,4 +46,27 @@ secbool __wur flash_erase(uint8_t sector);
 secbool __wur flash_write_byte(uint8_t sector, uint32_t offset, uint8_t data);
 secbool __wur flash_write_word(uint8_t sector, uint32_t offset, uint32_t data);
 
+typedef struct {
+  uint16_t first_sector;
+  uint16_t num_sectors;
+} flash_subarea_t;
+
+typedef struct {
+  flash_subarea_t subarea[4];
+  uint8_t num_subareas;
+} flash_area_t;
+
+const void *flash_area_get_address(const flash_area_t *area, uint32_t offset,
+                                   uint32_t size);
+uint32_t flash_area_get_size(const flash_area_t *area);
```

Comment:
Please remove `flash_area_get_size()`. Already declared in `storage/flash_common.h`.

Reply:
https://github.com/trezor/trezor-firmware/commit/626e6d0954357a79c03f7e8f80a947f9414f78f7